src/sage/categories/schemes.py

```diff
@@ -294,3 +296,27 @@ def extra_super_categories(self):
                     True
                 """
                 return [Rings()]
+
+
+class Jacobians(AbelianVarieties):
+    """
+    The category of Jacobians attached to curves or function fields.
+    """
```

Comment:
Do Jacobians have `additional_structure`?

Reply:
No. The doc of `additional_structure` says,
```
        A category `C` *defines additional structure* if `C`-morphisms
        shall preserve more structure (e.g. operations) than that
        specified by the super categories of `C`. 
```
As I see it, `Jacobians` category has as objects Jacobians, which are abelian varieties arisen from curves. Thus the morphisms between Jacobians are just morphisms of abelian varieties. 

An "additional structure" to be given to objects of the category `Jacobians` may be `base_curve()`, which gives the curve (or function fields that define curves abstractly) to which the Jacobian is attached. Other than this, I have no idea.

I defined new `Jacobians` category because I need to define functors from Jacobians to commutative groups. Currently neither `Jacobians` or `AbelianVarieties` has little functionality.

@nbruin may have more ideas.

Reply:
Perhaps I misunderstood your questions. 

Were you pointing that `class Jacobians(AbelianVarieties): ` is not the right way to define a subcategory? It should be `class Jacobians(Category_over_base): `? Then `class AbelianVarieties(Schemes_over_base):` is also wrong?

Do I also define `additional_strcuture()` even if there is no additional structure?

Reply:
> Were you pointing that `class Jacobians(AbelianVarieties): ` is not the right way to define a subcategory?

Yes, that was intended to be my main point

Reply:
> Should I also define `additional_structure()` even if there is no additional structure?

No

Reply:
I think `Schemes_over_base` is intended to be a "category type" just like `Category_over_base`. (I think that maybe the name was not chosen very carefully, and it should maybe have been called `SchemeCategory_over_base`.)
In any case, `AbelianVarieties` is defined correctly as a subclass of `Schemes_over_base`; and likewise `Jacobians` should be defined as `class Jacobians(Schemes_over_base)`. 
The subcategory relation is not expressed by subclassing; instead it is constructed dynamically by means of `super_categories`.

Reply:
Yes. I agree. I fixed it in https://github.com/sagemath/sage/pull/35467/commits/e8bd6dbed70fe2d1c3d8cec57be9967db3242589 and added abstract method `base_curve()` to the category.

Reply:
Maybe actually `AbelianVarieties` is not defined quite correctly. At least I can't make immediate sense of this:
```
sage: Schemes(ZZ).base_scheme()
Spectrum of Integer Ring
sage: AbelianVarieties(QQ).base_scheme()
Rational Field
```

Reply:
> I think `Schemes_over_base` is intended to be a "category type" just like `Category_over_base`. (I think that maybe the name was not chosen very carefully, and it should maybe have been called `SchemeCategory_over_base`.) 

It is also a genuine category (schemes over a base scheme). Hence the name `Schemes_over_base` is not bad.

On the other hand, I think it is wrong that `Schemes` category is a super category of `Schemes_over_base`. The category of vector spaces is not a subcategory of abelian groups, if I remember correctly the category theory.

Reply:
> Maybe actually `AbelianVarieties` is not defined quite correctly. At least I can't make immediate sense of this:
> 
> ```
> sage: Schemes(ZZ).base_scheme()
> Spectrum of Integer Ring
> sage: AbelianVarieties(QQ).base_scheme()
> Rational Field
> ```

Yes. That is inconsistent. It has its root in the usual abuse of language in scheme theory to say "S is over QQ" when "S is over Spec(QQ)". This would be better:
```
sage: Schemes(ZZ).base()
Integer Ring
sage: AbelianVarieties(QQ).base()
Rational Field
sage: Schemes(ZZ).base_scheme()
Spectrum of Integer Ring
sage: AbelianVarieties(QQ).base_scheme()
Spectrum of Rational Field
```

Reply:
I worked more on `Schemes` category.

Now 

1. `Schemes_over_base` category has bases in `Schemes` category.
2. `AbelianVarieties` and `Jacobians` categories have bases in `Fields` category.

Thus we have
```sage
sage: Schemes(ZZ).base()
Spectrum of Integer Ring
sage: Schemes(ZZ).base_scheme()
Spectrum of Integer Ring
sage: AbelianVarieties(QQ).base()
Rational Field
sage: AbelianVarieties(QQ).base_scheme()
Spectrum of Rational Field
sage: Jacobians(QQ).base()
Rational Field
sage: Jacobians(QQ).base_scheme()
Spectrum of Rational Field
```